lib/remote_page.py

```diff
@@ -1,11 +1,14 @@
+from bs4 import BeautifulSoup
```

Comment:
The imports were previously grouped correctly, but not anymore. Python built-in modules should be separated from third-party modules.

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
I only changed the order of the imports and not the grouping. Didn't even think about whether they should be grouped differently as well. Fixed in PR #943